### PR TITLE
AVRO-2441: Add quickstart guide for Python3

### PR DESCRIPTION
**Pull request by sekikn**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Avro Jira](https://issues.apache.org/jira/browse/AVRO/) issues and references them in the PR title. For example, "AVRO-1234: My Avro PR"
  - https://issues.apache.org/jira/browse/AVRO-2441
  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

No additional test since it's a documentation fix. I ran ant in the doc directory and confirmed that the output was as expected.

### Commits

- [x] My commits all reference Jira issues in their subject lines. In addition, my commits follow the guidelines from "[How to write a good git commit message](https://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain Javadoc that explain what it does

### CODE REVIEW

File: doc/src/content/xdocs/gettingstartedpython3.xml [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+        downloaded from the <a
+        href="https://avro.apache.org/releases.html">Apache Avro&#153;
+        Releases</a> page.  This guide uses Avro &AvroVersion;, the latest
+        version at the time of writing.  Download and unzip
+        <em>avro-python3-&AvroVersion;.tar.gz</em>, and install via <code>python3
+        setup.py</code> (this will probably require root privileges).  Ensure
+        that you can <code>import avro</code> from a Python prompt.
+      </p>
+      <source>
+$ tar xvf avro-python3-&AvroVersion;.tar.gz
```
Comment (Fokko) [inline doc/src/content/xdocs/gettingstartedpython3.xml]: Wouldn't it be easier to install it using pip?

Comment (sekikn) [reply doc/src/content/xdocs/gettingstartedpython3.xml]: Exactly! Just updated the PR.

File: doc/src/content/xdocs/gettingstartedpython3.xml [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+$ python3
+>>> import avro # should not raise ImportError
+      </source>
+      <p>
+        Or, it's published as <a href="https://pypi.org/project/avro-python3/">
+        the avro-python3 package</a> on <a href="https://pypi.org/">PyPI</a>,
+        so you can also use pip for installation:
+      </p>
+      <source>
+$ pip3 install avro-python3  # run as root if you want to install it globally
```
Comment (kojiromike) [inline doc/src/content/xdocs/gettingstartedpython3.xml]: Can we leave the "run as root" bit out? It's probably not the best advice and digging into the nuances of why is beyond the scope of this document.

Comment (sekikn) [reply doc/src/content/xdocs/gettingstartedpython3.xml]: Agreed, just updated the PR. Thanks for the comment!

File: doc/src/content/xdocs/gettingstartedpython3.xml [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+$ python3
+>>> import avro # should not raise ImportError
+      </source>
+      <p>
+        Alternatively, you may build the Avro Python library from source.  From
+        your the root Avro directory, run the commands
+      </p>
+      <source>
+$ cd lang/py3/
+$ sudo python3 setup.py install
```
Comment (jacobtolar) [inline doc/src/content/xdocs/gettingstartedpython3.xml]: If we [removed `sudo pip install`](https://github.com/apache/avro/pull/558#discussion_r296484784) perhaps `sudo python3 setup.py install` should also be updated? (You could suggest `setup.py install --user`?).

**Comment by sekikn** [comment]
Thank you for the review @jacobtolar! Just updated the setuptools' commands with `python setup.py install --user` for the consistency within this document.

**Comment by Fokko** [comment]
Great work @sekikn

**Pull request merged**